src/garage/_dtypes.py

```diff
@@ -548,6 +582,8 @@ class TimeStep(
         step_type (StepType): a :class:`~StepType` enum value. Can be one of
             :attribute:`~StepType.FIRST`, :attribute:`~StepType.MID`,
             :attribute:`~StepType.TERMINAL`, or :attribute:`~StepType.TIMEOUT`.
+        episode_info (dict): A dict of episode-level information.
```

Comment:
as with EpisodeBatch, please make this the first member/argument, and include the shape information. using the terminology from above, it would a dict of numpy arrays of shape `(S*^,)` (there is no batch dimension, because this is a single time step)